app/assets/javascripts/scp-igv.js

```diff
@@ -139,42 +139,74 @@ function initializeIgv() {
 
   const igvContainer = document.getElementById('igv-container')
 
-  const genes = $('.queried-gene')
-
   const genomeId = bamsToViewInIgv[0].genomeAssembly
 
   let reference
-  let locus
+  let searchOptions
+  let fallbackLocus
   if (genomeId === 'Macaca_fascicularis_5.0') {
```

Comment:
is this something that will get picked up in our analytics?  e.g. will we be able to answer the question "How many people are using IGV to view macaca genomes?"   Will we be able to answer the more general question of how often/what do people use IGV in SCP for?

Reply:
> will we be able to answer the question "How many people are using IGV to view macaca genomes?"

No, not yet.  SCP-2506 is a prerequisite.  That ticket has value beyond igv.js tracking.  This is the second time this question has been asked recently, so I've nominated the ticket for next sprint.

> Will we be able to answer the more general question of how often/what do people use IGV in SCP for?

_How often_ -- yup!

_What do people use it for_ -- not with analytics.  Qualitative interviews and value statements from users (e.g. original user report [here](https://broadinstitute.zendesk.com/agent/tickets/5718)) can answer this question more efficiently than quantitative analytics.  I'd love to set up some user interviews!